DiceKit/ProbabilityMass.swift

```diff
@@ -8,28 +8,36 @@
 
 import Foundation
 
-public struct ProbabilityMass: ApproximatelyEquatable {
+// TODO: Find a proper place for this
+public struct ProbabilityMassConfig {
```

Comment:
Size isn't a determining factor for me when choosing if a type needs to be extracted. So far any nested type for me has gone into a new file, because usually they need to conform to something, and then things get messy.

Besides for "moving" I was also talking about naming.